plugins/GrapesJsBuilderBundle/Assets/library/js/builder.service.js

```diff
@@ -305,6 +308,65 @@ export default class BuilderService {
       };
     }
   }
+
+  /**
+   * Extends GrapesJS using a JSON file provided by the current theme.
+   *
+   * The file has to be located in /themes/${Mautic.builderTheme}/grapejsblocks/
+   * The needs to be named:
+   * - page-html.json for the landing page editor
+   * - email-mjml.json for the MJML email editor
+   * - email-html.json for the HTML email editor
+   *
+   * The JSON should have two base properties: `blocks` and `types`.
+   *
+   * Example:
+   *
+   * ```js
+   * {
+   * 	"blocks": {
+   * 		"someSection": {
+   * 			// block definition here
+   * 		}
+   * 	},
+   * 	"types": {
+   * 		"aHeading": {
+   * 			// type definition here
+   * 		}
+   * 	}
+   * }
+   * ```
```

Comment:
Improve: add a link or mention that the structure of the object has to be compatible (or is fully compatible) with grapesjs blocks and types? So people who are not familiar with the GJS documentation know what the options are.